src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -2229,5 +2230,17 @@ def aks_trustedaccess_role_binding_create_or_update(cmd, client, resource_group_
     return client.create_or_update(resource_group_name, cluster_name, role_binding_name, roleBinding)
 
 
+def aks_trustedaccess_role_binding_update(cmd, client, resource_group_name, cluster_name, role_binding_name, roles):
+    TrustedAccessRoleBinding = cmd.get_models(
+        "TrustedAccessRoleBinding",
+        resource_type=CUSTOM_MGMT_AKS_PREVIEW,
+        operation_group="trusted_access_role_bindings",
+    )
+    existedBinding = client.get(resource_group_name, cluster_name, role_binding_name)
```

Comment:
what happens if the binding doesn't exist? should handle None case here?

Reply:
A similar situation is that in the `aks update` command CLI side does not handle None case. RP will return a `ResourceNotFound` error, and this error will be raised directly.

Reply:
Yes I noticed the `client.get` will raise NotFound error if the binding is notfound